_maps/map_files/PubbyStation/PubbyStation.dmm

```diff
@@ -47816,6 +47816,7 @@
 /obj/effect/turf_decal/stripes/line{
 	dir = 9
 	},
+/obj/structure/closet/secure_closet/miner,
```

Comment:
@Supermichael777 Why negative? That's the path of the unlocked locker that's present in every aux_base area.